src/libmusig2.c

```diff
@@ -72,17 +72,13 @@ static int musig2_calc_b(musig2_context_sig *mcs, musig2_param *param) {
     memcpy(&temp_concat[(1 + V) * XONLY_BYTES], param->msg, param->msg_len);
 
     /* Compute b */
-    return_val += !secp256k1_tagged_sha256(mcs->mc->ctx, param->b, tag, sizeof (tag), temp_concat, sizeof(temp_concat));
+    return secp256k1_tagged_sha256(mcs->mc->ctx, param->b, tag, sizeof (tag), temp_concat, sizeof(temp_concat));
```

Comment:
As with the calculation of `c`, this always returns 1, so we should set this function as `void`